README.md

```diff
@@ -294,6 +300,31 @@ Example:
 
 <br />
 
+#### Pagination
+
+The `pagination` property allows you to handle pagination on .
+
+Here's a list of variable names you may change:
+
+| Name   | Value                        | Description                                                  |
+| ------ | ---------------------------- | ------------------------------------------------------------ |
+| type   | `'buttons' | 'lazy-loading'` | Type of pagination. Buttons is the standard one. You can also have a "inifite scroll" with lazy loading. |
+| source | `'query'`                    | Where the pagination parameters are written to. Only supports query parameters for now. |
+| params | `object`                     | Parameters definition for pagniation purposes. See below.    |
```

Comment:
What about `fields`?

Reply:
I wasn't sure what I was going to do with them at first so I did not add them in the doc. Now I really only use `total`. I will remove others that will be useful for other types of pagination only.

Will add it now.